homeassistant/components/garages_amsterdam/sensor.py

```diff
@@ -0,0 +1,81 @@
+"""Sensor platform for Garages Amsterdam."""
+import logging
+
+from homeassistant.const import ATTR_ATTRIBUTION
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from . import get_coordinator
+from .const import ATTRIBUTION
+
+SENSORS = {
+    "free_space_short": "mdi:car",
+    "free_space_long": "mdi:car",
+    "short_capacity": "mdi:car",
+    "long_capacity": "mdi:car",
+}
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(hass, config_entry, async_add_entities):
+    """Defer sensor setup to the shared sensor module."""
+    coordinator = await get_coordinator(hass)
+
+    async_add_entities(
+        GaragesamsterdamSensor(coordinator, config_entry.data["garage_name"], info_type)
+        for info_type in SENSORS
+    )
+
+
+class GaragesamsterdamSensor(CoordinatorEntity):
+    """Sensor representing garages amsterdam data."""
+
+    unique_id = None
```

Comment:
```suggestion

```
I don't see a reason to have this defined here.

Reply:
If I omit this variable, no sensor entities will be created.

```
2020-11-16 18:05:33 ERROR (MainThread) [homeassistant] Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/home/klaas/core/homeassistant/helpers/entity_platform.py", line 302, in async_add_entities
    tasks = [
  File "/home/klaas/core/homeassistant/helpers/entity_platform.py", line 302, in <listcomp>
    tasks = [
  File "/home/klaas/core/homeassistant/components/garages_amsterdam/sensor.py", line 25, in <genexpr>
    GaragesamsterdamSensor(coordinator, config_entry.data["garage_name"], info_type)
  File "/home/klaas/core/homeassistant/components/garages_amsterdam/sensor.py", line 36, in __init__
    self.unique_id = f"{garage_name}-{info_type}"
AttributeError: can't set attribute
```